lit_llama/adapter_v2.py

```diff
@@ -26,20 +26,34 @@ def adapter_v2_state_from_state_dict(state_dict: dict) -> dict:
 
 
 def adapter_v2_new_forward(self, input: Tensor) -> Tensor:
+    weight = self.weight
+
+    try:
+        from lit_llama.quantization import Linear8bitLt
+        if isinstance(self, Linear8bitLt):
+            weight = self.dequantize(input.dtype)
+    except:
+        None
```

Comment:
It's more common to `pass`

```suggestion
        pass
```